tests/users/__snapshots__/test_data.ambr

```diff
@@ -493,6 +626,18 @@
     'primary_group': None,
   })
 # ---
+# name: TestUpdate.test_set_groups[pg_1]
+  <SQLUser(id='1', legacy_id='bf1b993c', active='True', administrator='False', b2c_display_name='', b2c_given_name='', b2c_family_name='', b2c_oid='', force_reset='False', handle='qtaylor', invalidate_sessions='False', last_password_change='datetime', primary_group_id='None')>
```

Comment:
Should these models include the `groups` field as well?

Reply:
SQLUser lazy loads the groups relationship, so groups will not show up in the snapshots